examples/README.md

```diff
@@ -2,9 +2,16 @@
 
 # Examples
 
-This directory contains fully functional code examples that you can use to learn more about Objax
+This directory contains fully functional code examples that you can use to learn more about Objax.
 
-See:
-* [Image Classification](classify/img/README.md)
+Examples from classic machine learning:
+* [Image Classification](image_classification/README.md)
+* [Text Generation](text_generation/README.md)
+
+Examples from recent research:
 * [Model-Agnostic Meta-Learning](optimization/README.md)
```

Comment:
I think you renamed `optimization` directory into `maml`.
Please update link here as well

Reply:
Done.